tests/stress/benchmarkstress_test.go

```diff
@@ -0,0 +1,90 @@
+package system
+
+import (
+	"strconv"
+	"testing"
+)
+
+const (
+	deployCommand = "while true ; do echo LONG RUNNING SERVICE ; sleep 5 ; done"
+)
+
+func BenchmarkStress25Environments0Deploys0Services0LoadBalancers(b *testing.B) {
+	benchmarkStress(25, 0, 0, 0, deployCommand, b)
+}
+func BenchmarkStress10Environments10Deploys0Services0LoadBalancers(b *testing.B) {
+	benchmarkStress(10, 10, 0, 0, deployCommand, b)
+}
+func BenchmarkStress20Environments20Deploys0Services0LoadBalancers(b *testing.B) {
+	benchmarkStress(20, 20, 0, 0, deployCommand, b)
+}
+func BenchmarkStress5Environments50Deploys0Services0LoadBalancers(b *testing.B) {
+	benchmarkStress(5, 50, 0, 0, deployCommand, b)
+}
+func BenchmarkStress5Environments100Deploys0Services0LoadBalancers(b *testing.B) {
```

Comment:
Deploys don't provide very meaningful stats IMO. I think we'd like to know some sort of max number we could support (e.g. 10k), but experience with Layer0 shows that this is rarely limiting factor compared to services, tasks, and environments.

Reply:
Removed lesser Deploy test.

✅